Extensions/PlatformBehavior/platformerobjectruntimebehavior.ts

```diff
@@ -965,17 +965,26 @@ namespace gdjs {
      * Update _potentialCollidingObjects member with platforms near the object.
      */
     private _updatePotentialCollidingObjects(maxMovementLength: float) {
+      const object = this.owner;
+
       this._manager.getAllPlatformsAround(
```

Comment:
Question: should the filtering be done by `getAllPlatformsAround`? Or by the RBush somehow?

Reply:
Yes, I moved it in `getAllPlatformsAround`.
RBush do what it is asked. It's not its fault it has outdated AABB.

Reply:
Perfect, thanks!